ci-operator/config/maistra/proxy/maistra-proxy-maistra-2.6.yaml

```diff
@@ -0,0 +1,166 @@
+base_images:
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.6"
+  use_build_cache: true
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 200m
+      memory: 200Mi
+tests:
+- as: unit
+  steps:
+    cluster_profile: ossm-aws
+    env:
+      CI: "true"
+      COMPUTE_NODE_REPLICAS: "1"
+      COMPUTE_NODE_TYPE: m5d.8xlarge
+      LOCAL_CPU_RESOURCES: "30"
+      LOCAL_JOBS: "30"
+      LOCAL_RAM_RESOURCES: "61440"
+      MAISTRA_BUILDER_IMAGE: registry.ci.openshift.org/ci/maistra-builder:2.6
+      MAISTRA_NAMESPACE: maistra-e2e-test
+      MAISTRA_SC_POD: maistra-proxy-test-pod
+    test:
+    - as: copy-src
+      cli: latest
+      commands: |
+        tar cf - . | oc exec -i -n "${MAISTRA_NAMESPACE}" "${MAISTRA_SC_POD}" -- tar xf - -C /work/
```

Comment:
do you need to mention the container name in the MAISTRA_SC_POD ?

Reply:
I have no idea. I just copied 2.5 jobs :/

Reply:
ok, I see. I forgot how I wrote those before